lib/Runtime/PlatformAgnostic/Platform/Linux/DateTime.cpp

```diff
@@ -160,7 +160,6 @@ namespace DateTime
         // tm doesn't have milliseconds
         int milliseconds = local->time % 1000;
 
-        tzset();
```

Comment:
> This won't have much of an impact since the other time functions like timelocal implicitly invoke tzset-like functionality when called.

@MSLaguana  That's why these are redundant hence this PR.